hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.tsx

```diff
@@ -23,6 +23,14 @@ import {
     GeneralOrderTitle,
 } from "components/general/OrderTables/OrderTables";
 import { Formik } from "formik";
+import { useDispatch, useSelector } from "react-redux";
+import { pendingOrderSelectors } from "slices/order/orderSlice";
```

Comment:
you are using the wrong slice here (sorry the names are the same, easy to be confused), but it should be imported from teamOrderSlice not orderSlice (orderSlice is for the participant side only), which is probably the reason for your problem.